lib/hydra/works/models/concerns/collection_behavior.rb

```diff
@@ -75,10 +75,6 @@ def parents
       member_of
     end
 
-    def in_collections
```

Comment:
I'm guessing because there's already an in_collections method defined by PcdmBehavior https://github.com/projecthydra-labs/hydra-pcdm/blob/master/lib/hydra/pcdm/models/concerns/pcdm_behavior.rb#L55-L57